masterlist.yaml

```diff
@@ -22304,10 +22336,16 @@ plugins:
         util: 'SSEEdit v4.0.3'
   - name: 'ClockworkSorting_Ing_CCCurios_CACO.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
+    msg:
+      - <<: *alreadyInOrFixedByX
+        subs: [ '"Additional Clockwork - Superior Sorting" (ClockworkSuperiorSorting.esp)' ]
+        condition: 'active("ClockworkSuperiorSorting.esp")'
     clean:
       - crc: 0x3297A568
         util: 'SSEEdit v4.0.3'
-  # Clockwork High poly Project Patches #
+      - crc: 0x30BAFDF7
+        util: 'SSEEdit v4.0.3'
```

Comment:
I get a different checksum, is this a future version of the plugin?

Reply:
Dang, you guys are _thurough_.
Nope, not for a future verison. It seems I must've been too tired to discern between the different sorting patches last night—this one's for ClockworkSorting_Ing_CCCurios (already in the masterlist)